tests/console/install_single_package.pm

```diff
@@ -36,6 +35,10 @@ sub run() {
 
     # ensure that package was installed correctly
     assert_script_run("rpm -q $pkgname");
+    assert_script_run("ci-openqa-tests");
```

Comment:
the name `tests/console/install_single_package.pm` does not relate to HPC but is generic so you should regard the content as such as well. If you do not want to provide test variables then at least you could just set your filenames as a default fallback. I don't see where my PR is leading in another direction.